<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Select which Items to import:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>281</width>
          <height>179</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QFrame" name="frame_6">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="1">
             <widget class="QListWidget" name="listWidget_Items">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>check an item to import it</string>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>GET_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GET_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PUT_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PUT_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CREATE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CREATE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CLOSE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CLOSE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>QSIZE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>QSIZE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MAXQSIZE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MAXQSIZE_END</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Items:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Example:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QListWidget" name="listWidget_Example">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>GET_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GET_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PUT_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PUT_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CREATE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CREATE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CLOSE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CLOSE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>QSIZE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>QSIZE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MAXQSIZE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MAXQSIZE_END</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_SelectAll">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Select all</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_SelectNone">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Select none</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_Reject">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_Accept">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
